### PR TITLE
[3.0] introduce SEO Schema Generator

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
         "rinvex/countries": "^6.0",
         "stof/doctrine-extensions-bundle": "^1.2",
         "symfony/symfony": "^4.4",
-        "webmozart/assert": "^1.2"
+        "webmozart/assert": "^1.2",
+        "spatie/schema-org": "^2.13"
```
Comment (jdreesen) [inline composer.json:39]: Careful, because this seems to need PHP `^7.3` while CoreShop is `^7.2`.

Comment (dpfaffenbauer) [reply composer.json:39]: thanks, tests also fail cause of that.

**Comment by solverat** [comment]
I'm not sure about this. This is basically the same as [dachcom-digital/pimcore-schema](https://github.com/dachcom-digital/pimcore-schema) does. We should consider removing the seo bundle and start using [dachcom-digital/pimcore-seo]/https://github.com/dachcom-digital/pimcore-seo) instead, to finally get rid of thousands of different seo implementation approaches.

This could be a good new feature for 3.0. Also: I also don't like the idea to use a request listener for all of this (Yea i know, i've done this by myself within the schema bundle...). Instead, it's way nicer to use a simple twig helper, as described [here](https://github.com/dachcom-digital/pimcore-seo/blob/master/docs/00_Setup.md#default-layout).

Join forces to build the last and final seo bundle forever is definitely something we need here, right? :) WDYT?

**Comment by dpfaffenbauer** [comment]
@solverat I agree, but CoreShop' SEO Bundle does more than just schema, that's why I added this here. Might be something for friends-of-pimcore?

**Comment by dpfaffenbauer** [comment]
decided against that, with 3.0 we gonna deprecate the SEO Bundle and use the dachcom bundle

**Pull request closed without merging**